.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
 
 cache:
     directories:
-        - .phpunit
+        - vendor/symfony/phpunit-bridge/bin/.phpunit
```

Comment:
Isn't it an "issue" that we write in the vendor directory? I'm pretty sure that we never write under vendor anywhere else.